File: sample/android/rn-sample-android-lib/src/main/java/com/example/rn_sample_android_lib/RNSampleApplication.java
package com.example.rn_sample_android_lib;

import android.app.Application;
import android.content.Context;

import androidx.multidex.MultiDex;

public class RNSampleApplication extends Application {

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        MultiDex.install(this);
    }
}

File: sample/android/rnsample/src/main/java/com/example/rnsample/RNSampleApplication.kt
package com.example.rnsample

import android.app.Application
import android.content.Context
import androidx.multidex.MultiDex

class RNSampleApplication : Application() {
    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }
}